content/en/synthetics/browser_tests.md

```diff
@@ -120,6 +121,8 @@ To create a variable, first give it a name then define its value from:
 | `{{ alphabetic(n) }}`   | Generates an alphabetic string with n letters.      |
 | `{{ alphanumeric(n) }}` | Generates an alphanumeric string with n characters. |
 
+* **An Email**: Generate a random Synthetics email address that you will use in your steps to assert on the email that was sent or to perform actions on the sent email (eg. click confirmation link).
```

Comment:
You should put this above the "A pattern" entry, in order to not have the table messing with the bullet list layout.

Reply:
Done!